addons/account/models/account_move.py

```diff
@@ -3242,6 +3242,23 @@ def action_send_and_print(self):
             'type': 'ir.actions.act_window',
         }
 
+    def _notify_get_recipients_groups(self, msg_vals=None):
+        groups = super()._notify_get_recipients_groups(msg_vals)
+        local_msg_vals = dict(msg_vals or {})
+        if self.move_type != 'entry':
+            for group_name, _group_method, group_data in groups:
```

Comment:
We should add something like this
```suggestion
            # This gives access to the document to the partners added to the emailing list in the send wizard
            for group_name, _group_method, group_data in groups:
```